include/cpp/jank/runtime/obj/transient_vector.hpp

```diff
@@ -23,6 +23,8 @@ namespace jank::runtime
       return ret;
     }
 
+    native_box<static_object> pop_in_place();
+
```

Comment:
This is a nit pick, but let's move this down below the other behaviors. Currently, the constructors go first, then the objectable behavior, followed by other behaviors, and then utilities.